src/azul/__init__.py

```diff
@@ -1590,6 +1592,32 @@ def docker_image_gists_path(self) -> Path:
 
     waf_rate_rule_limit = 1000
 
+    @attrs.frozen(auto_attribs=True, kw_only=True)
+    class FileDownloadLimit:
+        rate_limit: int
+        window_seconds: int
```

Comment:
```suggestion
        evaluation_window: int
```

Consistency, the other attributes don't encode the unit in their name either.